backend/test/models/riff_test.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# Tests for Riffs
+class RiffTest < ActiveSupport::TestCase
+  #   Assert that a Riff without a created_by (creator) is invalid.
+  test 'a Riff without a created_by (creator) is invalid' do
+    riff = build(:riff)
```

Comment:
I think you have to explicitly set the creator as nil like:

`riff = build(:riff, creator: nil)`